spec/lib/data_update_scripts/enable_admin_member_view_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20220225063600_enable_admin_member_view.rb",
+)
+
+describe DataUpdateScripts::EnableAdminMemberView do
+  after do
+    FeatureFlag.remove(:admin_member_view)
+  end
+
+  it "enables the :admin_member_view flag" do
+    expect do
+      described_class.new.run
+    end.to change { FeatureFlag.enabled?(:admin_member_view) }.from(false).to(true)
+  end
+
+  it "works if the flag is already available" do
+    FeatureFlag.add(:admin_member_view)
+
+    expect do
+      described_class.new.run
+    end.not_to change { FeatureFlag.exist?(:admin_member_view) }
+  end
```

Comment:
nit: I understand where you are coming from and how this is a small test file but it's generally a good idea to keep each test scenario isolated so the context is encapsulated and allows for random testing order.

Reply:
actually I misread, please ignore this.